.github/CODEOWNERS

```
@@ -3,4 +3,4 @@
 
 # These owners will be the default owners for everything in
 # the repo
-*       @benfterpay @ScottAntonacAP
+*       @benfterpay @ScottAntonacAP @BrianGardnerAtl @tir38 @pedronveloso @m-nogas
```

While you're here...
```suggestion
*       @ScottAntonacAP @BrianGardnerAtl @tir38 @pedronveloso @m-nogas
```